DecipherNow/gm-ui-components
# feat: use svg loader for icons

Note: I am still mocking svgs for our internal tests, which is why you might see "IconMock" in snapshots. This seems ok to me because all we really need to test is that Icons are receiving the correct props.

Relevant files to review:
- config/svgMock.js 
- config/svgr.config.js 
- config/template.js 
- package.json 
- webpack.config.js
- Icon.test.js

## To Test
- On this branch, run `npm run build && npm link`
- In gm-data-ui, run `npm uninstall gm-ui-components && npm link gm-ui-components`
- In the root of your app, import ThemeProvider and wrap the app, like so:
  ```javascript 
       import { ThemeProvider } from "styled-components"

       <ThemeProvider theme={{ brandColor: COLOR_BRAND_A }} >
            <App />
       </ThemeProvider>

- Check that Icons are rendering as expected
